ruby/ql/lib/codeql/ruby/dataflow/internal/DataFlowDispatch.qll

```diff
@@ -197,7 +197,7 @@ private predicate instanceMethodCall(CfgNodes::ExprNodes::CallCfgNode call, Modu
     or
     // When we don't know the exact type, it could be any sub class
     exact = false and
-    tp.getSuperClass+() = m
+    [tp.getSuperClass+(), tp.getAnIncludedModule+(), tp.getAPrependedModule+()] = m
```

Comment:
I guess it should really be
```
tp.getACludedModule+()
```
where `getACludedModule` is the union of `getAnIncludedModule` and `getAPrependedModule`?

Reply:
Perhaps even `getASuperCludedModule+` , a perhaps better named `getAnAncestor+`()?